OneBusAway/ui/info/OBASettingsViewController.m

```diff
@@ -42,6 +42,13 @@ - (void)loadData {
         [[NSUserDefaults standardUserDefaults] setBool:!analyticsValue forKey:OBAOptInToTrackingDefaultsKey];
     } switchValue:analyticsValue];
     [analyticsSection addRow:switchRow];
+    
+    // Disable review requests - issue #854
+    BOOL ratePromptValue = [[NSUserDefaults standardUserDefaults] boolForKey:OBADisallowReviewPromptsDefaultsKey];
+    OBASwitchRow *ratePromptRow = [[OBASwitchRow alloc] initWithTitle:NSLocalizedString(@"msg_disallow_review_prompt", @"A switch option's text for enabling and disabling App Store review prompts") action:^{
```

Comment:
So here you have a switch that when set to a positive value produces a negative result, right? On = Don't do something. Off = Do something. I find these negated settings hard to follow, and find this much easier to understand if the setting was, instead, for "Prompt me for app feedback", and was set to `YES` by default.

You can register default values in OBAApplication.m. Check out the `-registerAppDefaults` method.